sim/warrior/protection/protection_warrior.go

```diff
@@ -44,14 +44,14 @@ func NewProtectionWarrior(character core.Character, options proto.Player) *Prote
 		Options:  *warOptions.Options,
 	}
 
+	war.PseudoStats.CanBlock = war.Equip[proto.ItemSlot_ItemSlotOffHand].WeaponType == proto.WeaponType_WeaponTypeShield
```

Comment:
You shouldn't need to do this, it's already handled generically by `core`: https://github.com/wowsims/wotlk/blob/master/sim/core/character.go#L122